src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -706,10 +703,10 @@ Json::Value checkFee (
 
     if (fee > limit && fee != loadFee &&
         request.isMember("x-queue-okay") &&
-            request["x-queue-okay"].isBool() &&
+            request["x-queue-okay"].isConvertibleTo(Json::booleanValue) &&
```

Comment:
I agree that is `isConvertibleTo` is overly permissive. I get converting the string `"1"` or `"true"` or the integer `1` to the boolean `true`, but a real? Gah...